app/src/main/java/com/example/mytrainingpal/components/Navigation.kt

```diff
@@ -34,7 +34,7 @@ sealed class Screen(
         Screen(
             "musclePainMain",
             "Muscle Pain",
-            ShoulderPainIcon(),
```

Comment:
@raissacavalcanti Please undo this change. We want the shoulder pain icon instead of a sad smiley :)